### DISCUSSION TITLE
Raspberrymatic im LXC nur möglich als Snapshot

### DISCUSSION THREAD

**Discussion by sunnyecki1** [OpenCCU/OpenCCU#2671] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,

ich nutze Raspberrymatic seit einigen Jahren auf einem RPI3B. Das System läuft absolut stabil und zuverlässig. Parallel betreibe ich ioBroker auf einem RPI4.
Jetzt möchte ich beides zusammenführen auf einem RPI5 mit 8GB. Die Proxmox Installation auf dem RPI5 läuft gut und ich habe Raspberrymatic in einem LXC installiert.
Während der Installation konnte ich allerdings nur einen Snapshot der Version 3.75 auswählen, mir wurden keine stabilen Versionen angezeigt. Das System läuft mit dem Snaphot bisher auch gut, ich würde allerdings gerne wechseln zu einer stabilen Version, aus Gründen der Betriebssicherheit.
Wie aber kann ich das realisieren? 
Ich habe versucht die die VM/OVA Version zu installieren. Tatsächlich kann ich hier auch die 3.73 als stable auswählen. Das Image wird dann geladen, ich kann die Größe der Disk wählen und dann eine ID für die VM vergeben. Es kommt aber keine Auswahl für das Funkmodul. Ich verwende das HM-MOD-RPI-PCB. 
Kann es sein, dass dieses Modul nur mit der LXC Version läuft, aufgrund des direkten Zugriffs auf die GPIO oder sollte es eigentlich erkannt werden? 
Gäbe es eine Möglichkeit ein Downgrade im LXC vom Snapshot zum Release durchzuführen? 

Für ein paar hilfreiche Tipps wäre ich sehr dankbar.
Danke und viele Grüße aus Bremen
Jörg

**Comment by jens-maus** [comment, marked as answer]
> Die Proxmox Installation auf dem RPI5 läuft gut und ich habe Raspberrymatic in einem LXC installiert. Während der Installation konnte ich allerdings nur einen Snapshot der Version 3.75 auswählen, mir wurden keine stabilen Versionen angezeigt. Das System läuft mit dem Snaphot bisher auch gut, ich würde allerdings gerne wechseln zu einer stabilen Version, aus Gründen der Betriebssicherheit. Wie aber kann ich das realisieren?

Nun, die LXC Variante von RaspberryMatic bzw. die 3.75.x ist eben **noch** nicht als stable release freigegeben. Sieht man doch auch wenn man die offizielle Release Page sich einmal anschaut:

https://github.com/jens-maus/RaspberryMatic/releases

Da wird zum jetzigen Zeitpunkt als stable release die 3.73.x angeboten.

Du bist also schlichtweg der Zeit etwas voraus und so gibt es aktuell die LXC Installationsvariante lediglich als Development Snapshot und nicht als stable release.

Zu deiner Beruhigung kann ich jedoch sagen das sich das "Problem" jedoch wohl in den nächsten Tage erledigen sollte wenn eQ3 selbst die 3.75.x offiziell herausgibt wird dann auch RaspberryMatic recht bald danach auch basierend auf der 3.75.x freigegeben werden und dann auch mit der LXC Installationsvariante die es momentan eben auch nur als Development Snapshot gibt.

**Comment by sunnyecki1** [comment]
Moin moin,

vielen Dank für die schnelle Antwort. Ich war da etwas auf dem Holzwege. Mir war nicht bewusst, dass es als LXC Variante noch kein stable Release gibt. 
Also heißt es abwarten und dann auf die 3.75 stable aktualisieren.

Kurz noch die Frage zu meinem Funkmodul HM-MOD-RPI-PCB. Läuft das grundsätzlich auch in der VM/OVA Version und sollte ich die nutzen statt der LXC Version. Darüber gab es ja einige Diskussionen.

Viele Dank
Jörg
